moban/repo.py

```diff
@@ -60,7 +59,7 @@ def get_moban_home():
     from appdirs import user_cache_dir
```

Comment:
usercache:// is one choice and alternative, I could hide it in RepoFSOpener.

Reply:
The following moban file would works. IT is from level 10 use git as dependency.

```
requires:
  - https://github.com/moremoban/pypi-mobans
configuration:
  template_dir:
    - "usercache://moban:chfw/repos/pypi-mobans/templates"
    - local
  configuration: config.yml
  configuration_dir: "usercache://moban:chfw/repos/pypi-mobans/config"
targets:
  - mytravis.yml: travis.yml.jj2
  - test.txt: demo.txt.jj2

```

I must pass a user name for it to find moban, it may not work across the teams?